pkgs/development/tools/git-aggregator/default.nix

```
@@ -31,7 +32,7 @@ python3Packages.buildPythonApplication rec {
   '';
```

This appears to fix the build.
```suggestion
    git config --global pull.rebase false
  '';
```